Gaming The Syllabus/Assets/Scripts/Combat/UI/CombatUIManager.cs

```diff
@@ -87,6 +87,7 @@ private static Healthbar CreateHealthbar(CombatUnit unit, Vector2 position)
         Healthbar healthbar = healthbarGameObject.GetComponent<Healthbar>();
         healthbar.UpdateUnitName(unit.UnitName);
         healthbar.UpdateHealthbarValue(unit.CurrentHealth, unit.MaximumHealth);
+        healthbar.UpdateUnitSprite(unit.Sprite);
```

Comment:
ooo I'm assuming this will update the actual icon of the enemy during combat. Awesome to see.